js/app.js

```diff
@@ -210,6 +165,12 @@ function appendBlogTagToTagCloud(blogTag) {
 }
 
 
+function navigateToBlogs(blogTag) {
+    var urlToNavigate = location.protocol + '//' + location.host + location.pathname;
+    window.location.href = urlToNavigate + (blogTag ? "?tag=" + blogTag : "");
```

Comment:
just FYI there is also `location.replace` depending on whether you want this change to be navigable with Forward and Back: https://www.w3schools.com/howto/howto_js_redirect_webpage.asp

Reply:
Yes, I think it should be navigable